packages/appsync-emulator-serverless/__test__/example/schema.graphql

```diff
@@ -1,3 +1,5 @@
+scalar JSON
```

Comment:
Wrong type name: https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html#awsjson

This brings up the issue that we really will need to start building an integration suite for the emulator that validates our test schemas against real appsync to ensure we're not missing details like this.

Reply:
:( 

Good catch.

Hmmm, integration suite.  Solid idea.  How do you see it working?  Deploying the test example using serverless and ensuring the invocation succeeds?

Reply:
Thinking we write a test suite that works against both the emulator and appsync (via api key auth perhaps?)